packages/stark-ui/src/modules/minimap/components/_minimap-theme.scss

```diff
@@ -0,0 +1,196 @@
+/* ============================================================================== */
+/*  S t a r k   M i n i m a p                                                     */
+/* ============================================================================== */
+/* stark-ui: src/modules/minimap/components/minimap-theme.scss */
```

Comment:
The `theme.scss` file should contain only "themable" properties, meaning that only colors, backgrounds, fill colors, foregrounds, etc should be defined in this file

So let's move all the other CSS rules like top, left, right, etc to the `_minimap.component.scss` file ;)